code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -386,6 +383,11 @@
 	else
 		W.forceMove(get_turf(src))
 
+	for(var/mob/M in range(1, get_turf(src)))
```

Comment:
This was removed from before the movement of the object in question to after, so it wouldn't remove things un-necessarily.